client/src/pages/MainPage/MainPage.tsx

```diff
@@ -0,0 +1,8 @@
+import { Main } from '../../components/main';
+
+// needed:
+// separate component with all
```

Comment:
What do you mean? Separate component with all what?

Reply:
It was my side note to create separate component with things that will collect data in the future. Basically just for readability.